docs/README.md

```diff
@@ -11,5 +11,6 @@ Earth Enterprise Fusion & Server currently run on 64-bit versions of the followi
 * Red Hat Enterprise Linux versions 6.0 to 7.2, including the most recent security patches
 * CentOS 6.0 to 7.2
 * Ubuntu 10.04, 12.04 and 14.04 LTS
+* Mint 18.2
```

Comment:
I'm not sure if we want to officially state that we support Mint since that implies we will test every release on it.  It might be better to leave it off even though it probably works fine.

Reply:
Aw ok ..  Is it ok to leave it in the scripts since it doesn't affect any other users ?

Reply:
Yes, I don't have any problem with leaving it in the scripts.